pkg/cmd/common.go

```diff
@@ -21,8 +21,7 @@ func setupClient(c *cli.Context) (*client.Client, *config.EnvConfig, error) {
 	if err := cfg.Load(); err != nil {
 		return nil, nil, err
 	}
-
-	endpoint := c.GlobalString("endpoint")
+	endpoint := c.String("endpoint")
```

Comment:
In v1 we had to access the global flag namespace because the `endpoint` flag was defined in the root command -- is that no longer the case? We should test this change.

Reply:
@coryschwartz ?

Reply:
Yeah this still works, i.e.
```
testground --endpoint 192.168.1.50:8081 healthcheck --runner local:docker
Apr 28 16:03:13.267328	INFO	using home directory: /home/cory/testground
Apr 28 16:03:13.267750	INFO	testground client initialized	{"addr": "192.168.1.50:8081"}